google/cloud/bigquery/_tqdm_helpers.py

```diff
@@ -95,19 +110,15 @@ def wait_for_query(
             current_stage = query_job.query_plan[i]
             progress_bar.total = len(query_job.query_plan)
             progress_bar.set_description(
-                "Query executing stage {} and status {} : {:0.2f}s".format(
-                    current_stage.name,
-                    current_stage.status,
-                    time.time() - start_time,
-                ),
+                f"Query executing stage {current_stage.name} and status {current_stage.status} : {time.perf_counter() - start_time:.2f}s"
             )
         try:
             query_result = query_job.result(
                 timeout=_PROGRESS_BAR_UPDATE_INTERVAL, max_results=max_results
             )
             progress_bar.update(default_total)
             progress_bar.set_description(
-                "Query complete after {:0.2f}s".format(time.time() - start_time),
+                f"Job ID {query_job.job_id} successfully executed",
```

Comment:
Might want to include the location in the output too, formatted like the bq cli as location:I'd https://cloud.google.com/python/docs/reference/bigquery/latest/google.cloud.bigquery.job.QueryJob#google_cloud_bigquery_job_QueryJob_location

I believe the location is required to fetch the job metadata for the non-default location.